<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectorWindow</class>
 <widget class="QMainWindow" name="DirectorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>717</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>717</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>教学大纲管理系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>115</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QPushButton" name="btn_confirm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>确认修改</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_show">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>查看分配</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>85</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>系主任界面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_exit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>380</y>
      <width>451</width>
      <height>43</height>
     </rect>
    </property>
    <widget class="QComboBox" name="box_permission">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>11</y>
       <width>87</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>增加权限</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>取消权限</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="box_course">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>11</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="box_ID">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>11</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>149</x>
      <y>10</y>
      <width>551</width>
      <height>371</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>549</width>
       <height>369</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>课程</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>负责人ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>负责人姓名</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string> 选择操作</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
